### ISSUE TITLE
ClumsySyntax - missing error with sonarlint

### ISSUE THREAD

**Issue by clement-brodu** [Riverside-Software/sonar-openedge#1122]
Hello,

Sonarlint (in eclipse) doesn't raise an issue in the case above.

The same code raise an error with sonar-scanner / sonarqube.

Version :

* Openedge 12.8.2
* Plugin 2.26.0
* Sonarlint 9.3.3006

```
INTERFACE Sonar.ClumsySyntaxeInterface:  

    METHOD PUBLIC VOID Toto(  ): // Should raise error because of ":" instead of "."

END INTERFACE.
```

**Comment by gquerret** [comment]
Reproduced. Rules included in the standard plugin (like ClumsySyntax), or in the security plugin are not executed in this version of SonarLint. That will be fixed.

**Comment by gquerret** [comment]
Fixed in develop branch, will be included in 2.30
